src/qibolab/instruments/qblox.py

```diff
@@ -626,14 +626,80 @@ def _get_next_sequencer(self, port, frequency, qubit):
+            "cont_mode_waveform_idx_awg_path1",
+            "demod_en_acq",
+            "gain_awg_path0",
+            "gain_awg_path1",
+            "integration_length_acq",
+            "marker_ovr_en",
+            "marker_ovr_value",
+            "mixer_corr_gain_ratio",
+            "mixer_corr_phase_offset_degree",
+            "mod_en_awg",
+            "nco_freq",
+            "nco_phase_offs",
+            "nco_prop_delay_comp",
+            "nco_prop_delay_comp_en",
+            "offset_awg_path0",
+            "offset_awg_path1",
+            "sync_en",
+            # "thresholded_acq_marker_address",
+            "thresholded_acq_marker_en",
+            "thresholded_acq_marker_invert",
+            "thresholded_acq_rotation",
+            "thresholded_acq_threshold",
+            # "thresholded_acq_trigger_address",
+            "thresholded_acq_trigger_en",
+            "thresholded_acq_trigger_invert",
+            "trigger10_count_threshold",
+            "trigger10_threshold_invert",
+            "trigger11_count_threshold",
+            "trigger11_threshold_invert",
+            "trigger12_count_threshold",
+            "trigger12_threshold_invert",
+            "trigger13_count_threshold",
+            "trigger13_threshold_invert",
+            "trigger14_count_threshold",
+            "trigger14_threshold_invert",
+            "trigger15_count_threshold",
+            "trigger15_threshold_invert",
+            "trigger1_count_threshold",
+            "trigger1_threshold_invert",
+            "trigger2_count_threshold",
+            "trigger2_threshold_invert",
+            "trigger3_count_threshold",
+            "trigger3_threshold_invert",
+            "trigger4_count_threshold",
+            "trigger4_threshold_invert",
+            "trigger5_count_threshold",
+            "trigger5_threshold_invert",
+            "trigger6_count_threshold",
+            "trigger6_threshold_invert",
+            "trigger7_count_threshold",
+            "trigger7_threshold_invert",
+            "trigger8_count_threshold",
+            "trigger8_threshold_invert",
+            "trigger9_count_threshold",
+            "trigger9_threshold_invert",
+            "upsample_rate_awg_path0",
+            "upsample_rate_awg_path1",
+        ]
+
         # select a new sequencer and configure it as required
         next_sequencer_number = self._free_sequencers_numbers.pop(0)
         if next_sequencer_number != self.DEFAULT_SEQUENCERS[port]:
-            for parameter in self.device.sequencers[self.DEFAULT_SEQUENCERS[port]].parameters:
+            for parameter in parameters:
                 value = self.device.sequencers[self.DEFAULT_SEQUENCERS[port]].get(param_name=parameter)
-                if not value is None:
-                    target = self.device.sequencers[next_sequencer_number]
-                    self._set_device_parameter(target, parameter, value=value)
+                if value is None:
+                    raise RuntimeError
```

Comment:
Please raise errors including text explaining the problem.

Reply:
Sorry, this was my own test to verify if any of the values were none, but they weren't.
There is no need to raise an error, I have replaced it with:
```python
                if value:
                    target = self.device.sequencers[next_sequencer_number]
                    self._set_device_parameter(target, parameter, value=value)
``` 
This way, if the value of one parameter is None, it continues with the next.